attributes/default.rb

```diff
@@ -77,11 +80,10 @@
 default['haproxy']['frontend_ssl_max_connections'] = 2000
 
 default['haproxy']['install_method'] = 'package'
-default['haproxy']['conf_dir'] = '/etc/haproxy'
 
-default['haproxy']['source']['version'] = '1.6.9'
-default['haproxy']['source']['url'] = 'http://www.haproxy.org/download/1.6/src/haproxy-1.6.9.tar.gz'
-default['haproxy']['source']['checksum'] = 'cf7d2fa891d2ae4aa6489fc43a9cadf68c42f9cb0de4801afad45d32e7dda133'
+default['haproxy']['source']['version'] = '1.6.11'
```

Comment:
Should we bump this to 1.7.X since it's stable now and just considered a cleaned up 1.6 release?